tests/setuponlytests/test.sh

```diff
@@ -40,7 +40,7 @@ $logs
     echo "${folder} PASS"
   fi
 
-  docker-compose down -v --remove-orphans
+  docker-compose down -v --remove-orphans > /dev/null
```

Comment:
Do we need the -v if we are just outputting to Null anyways? Notting seeing -v referenced, but pretty sure it's short for --verbose. https://docs.docker.com/compose/reference/

Reply:
It's for removing any names volumes

https://docs.docker.com/compose/reference/down/

Reply:
Doh, I think I remember that now. :) Glad you caught it!